src/config.c

```diff
@@ -265,6 +263,46 @@ unsigned int parse_num_arg (const char *arg, const char *arg_name)
 	return result;
 }
 
+int parse_webpa_url_a (const char *full_url, 
+	char **server_addr, unsigned int *port)
+{
+  int allow_insecure;
+  unsigned int port_val;
+  int buflen = strlen (full_url) + 1;
+  char *url_buf = NULL;
+  char port_buf[8];
+
+#define ERROR_RTN(msg) \
```

Comment:
It's dangerous if the macro is defined far away from its usage, and the name doesn't indicate obviously what it does.
In this case, the name 'ERROR_RTN' makes it pretty obvious.
It's defined within a function, within a few lines of its usage, and it is #undef'd at the end of the function, so it can't hurt anybody.